docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -2435,9 +2435,17 @@ alertmanager_client:
 
 # (experimental) Enable UTF-8 strict mode. Allows UTF-8 characters in the
 # matchers for routes and inhibition rules, in silences, and in the labels for
-# alerts. It is recommended to check both alertmanager_matchers_disagree_total
-# and alertmanager_matchers_incompatible_total metrics before using this mode as
-# otherwise some tenant configurations might fail to load.
+# alerts. It is recommended that all tenants run the `migrate-utf8` command in
+# mimirtool before enabling this mode as otherwise some tenant configurations
+# might fail to load. Tenants with incompatible configurations can be identified
+# in Mimir server logs searching for lines containing `Alertmanager is moving to
+# a new parser for labels and matchers, and this input is incompatible.`. Tenant
+# configurations that are valid but contain ambiguous matchers can be found
+# searching for log lines containing `Matchers input has disagreement`. Each log
+# line includes the invalid input, a suggestion on how to fix the input, and the
+# ID of the affected tenant. Mimir must be run with debug level logging enabled.
+# More information about this can be found at
```

Comment:
```suggestion
# ID of the affected tenant. To use this mode, you must run Mimir with debug-level logging enabled.
# For more information, refer to
```

Reply:
Nit: Debug level logging must be enabled to find affected tenants, not use the mode. Does that change the phrasing at all?

Reply:
Yes, thank you for commenting. Updated my suggestion :)